app/controllers/admin/loan_questions_controller.rb

```diff
@@ -72,9 +72,12 @@ def set_loan_question
 
     # Never trust parameters from the scary internet, only allow the white list through.
```

Comment:
Can we please delete this comment. The *_params method is a Rails convention, so not needed.